libs/sdk-model/src/theme/index.ts

```diff
@@ -0,0 +1,140 @@
+// (C) 2020 GoodData Corporation
```

Comment:
we have since refined/reviewed the logic to distribute contents between model and backend-spi. newly, these types should be in backend-spi.